doc/source/whatsnew/v0.19.0.txt

```diff
@@ -361,6 +361,8 @@ Google BigQuery Enhancements
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- The ``.get_credentials()`` method of ``GbqConnector`` can now first try to fetch the application default credentials (https://developers.google.com/identity/protocols/application-default-credentials). This is only valid if the parameter ``private_key`` is not provided. Using this functionality, the ``OAuth2WebServerFlow`` call can be skipped (:issue:`13577`).
```

Comment:
put the URL as markdown e.g. ``[text here](url here)``. Also you might want to add this url to the documentation itself in io.rst.